modules/040-node-manager/hooks/util.go

```diff
@@ -133,6 +141,8 @@ func buildUpdateStatusPatch(
 		"statusMessage": statusMsg,
 	}
 
+	patch["conditions"] = conditionsToPatch(newConditions)
```

Comment:
Couldn't we just send an array here? `conditions` field is an array, why do we need the conversion?

Reply:
Yes, we need to add conversion because it is patch and we need to pass map[string]interface{} object for correct conversation